src/components/carousel/carousel.styles.ts

```diff
@@ -0,0 +1,131 @@
+import { css } from 'lit';
+import componentStyles from '../../styles/component.styles';
+
+export default css`
+  ${componentStyles}
+
+  :host {
+    display: flex;
+
+    --gutters: var(--sl-spacing-medium);
+    --slide-width: 100%;
+    --slide-height: 100%;
+    --slide-aspect-ratio: 3 / 2;
+    --arrows-size: var(--sl-font-size-2x-large);
+    --scroll-hint-margin: 0;
+  }
+
+  .carousel {
+    min-height: 100%;
+    min-width: 100%;
+
+    display: grid;
+
+    gap: var(--sl-spacing-medium);
+
+    grid-template-columns: min-content 1fr min-content;
+    grid-template-rows: 1fr min-content;
+    grid-template-areas:
+      '. slides .'
+      '. pagination .';
+
+    align-items: center;
+
+    position: relative;
+  }
+
+  .carousel__pagination {
+    grid-area: pagination;
+
+    display: flex;
+    justify-content: center;
+    gap: var(--sl-spacing-small);
+  }
+
+  .carousel__slides {
+    grid-area: slides;
+
+    width: 100%;
+    height: 100%;
+
+    display: grid;
+    grid-auto-columns: 100%;
+    grid-auto-rows: 100%;
+    grid-auto-flow: column;
+    align-items: center;
+
+    gap: var(--gutters);
+
+    overflow: auto;
+    scroll-snap-type: x mandatory;
+    overscroll-behavior-x: contain;
+    scrollbar-width: none;
+
+    padding-inline: var(--scroll-hint-margin);
+
+    aspect-ratio: var(--slide-aspect-ratio);
+  }
+
+  .carousel__slides::-webkit-scrollbar {
+    display: none;
+  }
+
+  .carousel__arrows {
+    grid-area: arrows;
+    display: contents;
+
+    font-size: var(--arrows-size);
+  }
+
+  .carousel__arrow {
+    flex: 0 0 auto;
+    display: flex;
+    align-items: center;
+    background: none;
+    border: none;
+    border-radius: var(--sl-border-radius-medium);
+    font-size: inherit;
+    color: var(--sl-color-neutral-600);
+    padding: var(--sl-spacing-x-small);
+    cursor: pointer;
+    transition: var(--sl-transition-medium) color;
+    appearance: none;
+  }
+
+  .carousel__arrow--disabled {
+    opacity: 0.5;
+    cursor: not-allowed;
+  }
+
+  .carousel__arrow--disabled::part(base) {
+    pointer-events: none;
+  }
+
+  .carousel__arrow--previous {
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .carousel__arrow--next {
+    grid-column: 3;
+    grid-row: 1;
+  }
+
+  .carousel__paginationItem {
+    display: block;
+    cursor: pointer;
+    background: none;
+    border: 0;
+    border-radius: var(--sl-border-radius-circle);
+    width: var(--sl-spacing-small);
+    height: var(--sl-spacing-small);
+    background-color: var(--sl-color-neutral-300);
+    will-change: transform;
+    transition: var(--sl-transition-fast) ease-in;
+  }
+
+  .carousel__paginationItem--active {
+    background-color: var(--sl-color-neutral-600);
+    transform: scale(1.2);
```

Comment:
Is this still being used? I don't see it in any of the examples, but maybe I missed it.

Reply:
It's used to for active dots in the pagination (see `renderPagination` method)